env0/resource_environment.go

```diff
@@ -40,6 +40,11 @@ func resourceEnvironment() *schema.Resource {
 	}
 }
 
+var VariableTypes = map[string]client.ConfigurationVariableType{
```

Comment:
can't make it constant :(
https://groups.google.com/g/golang-nuts/c/xUaDMciyB5Q